simulation: True
visualization: True
FOV: 1.57
min_footprint: 2
speed: 150
area_width: 100.0
area_height: 100.0
area_rotation: 0.0
logging: false
log_folder: /home/abbas/ros_workspace/src/active_survey/log
auto_exit: true
bypass_controller: true
prior_file_name: priors/prior10.jpg
time_limit: 1800
turning_time: 5.0
gp_sigma: 0.1
coarse_coverage_height: 10.0
sensing_height: 2.0
average_speed: 1.0
uncertain_cells_threshold: 1.0
discount_factor: 0.999
percent_interesting: 25
random_seed: 18201
patches: 9
policy: greedy
exploitation_rate: 1.005
